content/en/docs/setup/production-environment/tools/kubeadm/install-kubeadm.md

```diff
@@ -27,8 +27,7 @@ see the [Creating a cluster with kubeadm](/docs/setup/production-environment/too
 * Unique hostname, MAC address, and product_uuid for every node. See [here](#verify-mac-address) for more details.
 * Certain ports are open on your machines. See [here](#check-required-ports) for more details.
 * Swap configuration. The default behavior of a kubelet was to fail to start if swap memory was detected on a node.
-  Swap has been supported since v1.22. And since v1.28, Swap is supported for cgroup v2 only; the NodeSwap feature
-  gate of the kubelet is beta but disabled by default.
+  See [Swap memory management](/docs/concepts/architecture/nodes/#swap-memory) for more details.
```

Comment:
Additionally, we might need to revise the [NodeSwap](https://kubernetes.io/docs/reference/command-line-tools-reference/feature-gates/#feature-gates-for-alpha-or-beta-features) feature gate [description file (here)](https://github.com/kubernetes/website/blob/main/content/en/docs/reference/command-line-tools-reference/feature-gates/node-swap.md?plain=1) to include another 'beta' stage, to fully complement and complete this change. 

[Code reference (link here)](https://github.com/kubernetes/kubernetes/blob/0130072b053f85fb736c24d34552208cdd1bccfe/pkg/features/kube_features.go#L559-L563)

```
   // alpha: v1.22
	// beta1: v1.28 (default=false)
	// beta2: v.1.30 (default=true)

	// Permits kubelet to run with swap enabled.
	NodeSwap featuregate.Feature = "NodeSwap"
```